Tensile/TensileCreateLibrary.py

```diff
@@ -785,6 +788,7 @@ def writeKernels(
     prepAsm(
         kernelWriterAssembly,
         os.name != "nt",
+        # Use globalParameters here, not params
```

Comment:
I don't understand this comment given the context of the diff.

Reply:
There could be potentially two ways to access the data needed for prepAsm, one through globalParameters, the other through params. This is a quick comment to help explain why globalParameters is used here (which it should be because the global path is manipulated during the lifetime of this function).